### PR TITLE
Use and update MetaTagSocialMedia

### PR DESCRIPTION
**Pull request by SukhOberoi**
### Contributor checklist

- [x] This pull request is on a [separate branch](https://docs.github.com/en/get-started/quickstart/github-flow) and not the main branch

---

### Description
- Updated props for MetaTagSocialMedia component
- Used MetaTagSocialMedia in LandingCommunityBanner and Footer
- Updated MetaTagSocialMedia referenced in ModalSharePage
- Double-Checked that current designs are consistent

### Related issue

- #750

### CODE REVIEW

**Comment by andrewtavis** [comment]
Hey there @SukhOberoi 👋 Looks like some of the files for the frontend that you changed need to be reformatted. Could you install the Prettier VS Code extension (if you use VS Code) or pre-commit? pre-commit is detailed in the contributing guide and will check your commits before they're sent. Format those files and send along another commit, then I can review 😊

**Comment by SukhOberoi** [comment]
Hi. Could you check the new commit?

**Comment by andrewtavis** [comment]
Hi @SukhOberoi, I've been sick over the last few days. I'll try to get to this soon :)

**Comment by SukhOberoi** [comment]
Sure. Take your time. Get well soon 🥰

File: frontend/components/footer/flex/FooterFlexCol.vue [hunk 1]
```diff
@@ -48,10 +48,12 @@
...
             >
-              <Icon :name="connect.iconName" class="text-2xl sm:text-base" />
-              <p class="sr-only sm:not-sr-only">
-                {{ $t(connect.name) }}
-              </p>
+              <MetaTagSocialMedia
+                class="text-2xl sm:text-base"
+                :iconName="connect.iconName"
+                :text="$t(connect.name)"
+                textStyleClass="sr-only sm:not-sr-only"
```
Comment (andrewtavis) [inline frontend/components/footer/flex/FooterFlexCol.vue:55]: I'll rename the prop to `textClassUtilities` as it's a bit more descriptive of Tailwind classes. As it stands it looks more like it's for styling the text like passing color or size props, which isn't what we're using it for as of now 😊 Really great use of props though, @SukhOberoi!

File: frontend/components/landing/LandingCommunityBanner.vue [hunk 2]
```diff
@@ -6,34 +6,40 @@
       class="flex justify-center text-lg space-x-12 sm:space-x-8 xl:space-x-12 xl:text-xl"
     >
       <a
-        class="flex items-center space-x-3 hover:text-light-text dark:hover:text-dark-text focus-brand"
+        class="flex items-center space-x-2 hover:text-light-text dark:hover:text-dark-text focus-brand"
```
Comment (andrewtavis) [inline frontend/components/landing/LandingCommunityBanner.vue:9]: We don't need `flex items-center space-x-2` here as this is taken care of by the component now :)

File: frontend/components/meta-tag/MetaTagSocialMedia.vue [hunk 3]
```diff
@@ -1,9 +1,7 @@
 <template>
-  <div
-    class="flex content-center w-full h-full cursor-pointer gap-2 text-light-text dark:text-dark-text hover:text-light-distinct-text dark:hover:dark-distinct-text"
```
Comment (andrewtavis) [inline frontend/components/meta-tag/MetaTagSocialMedia.vue:3]: Nice job removing the colors and letting them be defined usage-wise!

**Review by andrewtavis — APPROVED** [review]
The commit above fixes some of the minor things that I detailed in the comments, @SukhOberoi 😊 Really really great execution here! Appreciate the care you put into this, as well as the quick reaction to the initial comments 🚀

**Pull request merged**